explorer/ast/bindings.cpp

```diff
@@ -37,21 +37,33 @@ void Bindings::Add(Nonnull<const GenericBinding*> binding,
   }
 }
 
-auto Bindings::None() -> Nonnull<const Bindings*> {
-  static Nonnull<const Bindings*> bindings = new Bindings;
-  return bindings;
-}
-
 void Bindings::Print(llvm::raw_ostream& out) const {
-  out << "{";
+  std::vector<std::pair<Nonnull<const GenericBinding*>, Nonnull<const Value*>>>
+      args;
+
+  for (const auto& [binding, value] : args_) {
+    args.push_back({binding, value});
+  }
+
+  std::stable_sort(args.begin(), args.end(), [](auto& a, auto& b) {
+    return a.first->index() < b.first->index();
+  });
+
   llvm::ListSeparator sep;
-  for (const auto& [name, value] : args_) {
-    out << sep << *name << " -> " << *value;
+  out << " >  bindings args: [";
+  for (const auto& [binding, value] : args) {
+    out << sep << "`" << *binding << "`: `" << *value << "`";
   }
-  for (const auto& [name, value] : witnesses_) {
-    out << sep << *name << " -> " << *value;
+  out << "]\n >  bindings witnesses: [";
+  for (const auto& [binding, value] : witnesses_) {
```

Comment:
You can use `binding->type_var()->index()` as the sort key here.

Reply:
Okay, I'll do that in next commit.